crates/test-utils/Cargo.toml

```diff
@@ -13,11 +13,17 @@ license.workspace = true
 
 [dependencies]
 kakarot-rpc-core={ path = "../core" }
+kakarot-rpc= { path = "../eth-rpc" }
```

Comment:
I thought that the test utils weren't supposed to depend on the RPC any more

Reply:
@ClementWalter I can remove this dependency, but that means moving [kakarot_rpc::test_utils](https://github.com/kkrt-labs/kakarot-rpc/blob/main/crates/eth-rpc/src/test_utils.rs).

So, if you think that will work then lemme know!

Reply:
@ClementWalter as discussed in the chat, I don't think we should move it back, we need a better refactor of the code to avoid this!

We can move ahead with it, but then later on focus to refactor the codebase so that such coupling doesn't exist.